Server.js

```diff
@@ -6,18 +6,21 @@ var winston = require('winston');
 var fs = require('fs');
 var tcpPortUsed = require('tcp-port-used');
 var http = require('http');
-var https = require('https');
 
-try {
-  fs.statSync('./config/config.js');
-} catch (error) {
-  console.log("Config file is not setup. Read README.md for config setup.");
-  return;
-}
-var config = require('./config/config');
-var models = require('./models');
-var log = require('./logging');
+var configPath = './config'
+process.argv.forEach((val, index) => {
+  if (val.toLowerCase().startsWith('config=')) {
+    configPath  = val.split('=')[1];
+  }
+});
+
+var allConfig = require('./cacconfig');
```

Comment:
Can `cacconfig` just be called `config`?

Also, why assign the imported cacconfig module to a variable named `allConfig`? That's a bit confusing.